src/chainstate/stacks/db/blocks.rs

```diff
@@ -3875,10 +3876,12 @@ impl StacksChainState {
         for microblock in microblocks.iter() {
             debug!("Process microblock {}", &microblock.block_hash());
             for tx in microblock.txs.iter() {
-                let (tx_fee, tx_receipt) =
+                let (tx_fee, mut tx_receipt) =
                     StacksChainState::process_transaction(clarity_tx, tx, false)
                         .map_err(|e| (e, microblock.block_hash()))?;
 
+                // TODO: also keep track of a real tx_index here and add to receipt
```

Comment:
Does this `TODO` need to be addressed before merge?

Reply:
@zone117x I remember you mentioning we need the tx indices of tx's within microblocks? Is that still true?

Reply:
Let's just make an issue for this so we can merge it.